cmd/main.go

```diff
@@ -183,34 +177,34 @@ func loadConfigs() (*config.Config, error) {
 		log.Println(fmt.Sprintf("Configuration loaded from environment, initial %s saved", configFile))
 		return cfg, nil
 	}
-	err := loadCertificate(cfg)
+	err := loadCertificate(&cfg.MQTT)
 	if err != nil {
 		return cfg, err
 	}
 	log.Println(fmt.Sprintf("Configuration loaded from file %s", configFile))
 	return cfg, nil
 }
 
-func loadCertificate(cfg *config.Config) error {
+func loadCertificate(cfg *config.MQTTConf) error {
 
```

Comment:
Please remove the empty line.

Reply:
@mteodor please fix this so we can progress.